src/moda_bitbuf.F

```diff
@@ -9,20 +10,16 @@
 
 	MODULE MODA_BITBUF
 
-C>        @var maxbyt
 C>        Maximum length of an output BUFR message.
-C>
-C>        @var mbyt
-C>        Length (in bytes) of current BUFR message for each internal
-C>        I/O stream.
-C>
-C>        @var mbay
-C>        Current BUFR message for each internal I/O stream.
-
 	  INTEGER :: MAXBYT
```

Comment:
Interesting that it seems the \@var tag isn't really needed here.  Per https://doxygen.nl/manual/commands.html#cmdvar, I had thought that tag was needed to get the variable display shown in the [original HTML output](https://emc.ncep.noaa.gov/emc/pages/infrastructure/bufrlib_jeff/html_new/namespacemoda__bitbuf.html#a53837bd62186cdc39c8062a88e6489d2), but it looks like you get similar results in the [new HTML output](https://emc.ncep.noaa.gov/emc/pages/infrastructure/bufrlib_jeff/html_PR253/namespacemoda__bitbuf.html#aeb00e574adb63bf7cc809d540d5a02bc) even if it's not there.    Good to know!